book/website/community-handbook/coworking/coworking-dropins.md

```diff
@@ -0,0 +1,90 @@
+# Drop-in Sessions
+
+Drop-in sessions are an opportunity for a team or group of people to host regular opportunities for other people within the community or organisation to ask questions or start a conversation on a topic of their interest.
+
+At the Turing, these include the [Turing Infrastructure Drop-in Sessions](https://github.com/alan-turing-institute/infrastructure-drop-ins) (online and in person), the TREx Virtual Drop in Session (online only), the Turing Research Ethics (TREx) Virtual Drop in Session (online only), and the Data Protection virtual drop in (online only).
+
+These sessions typically include an online component, typically using [Zoom](https://www.zoom.us/). 
+They may follow a different format or sign up person, depending on the needs and preferences of the organising team and the target community the sessions are designed for.
+
+The primary goal of these sessions is to provide an opportunity for answering targeted questions on a subject that the team holds expertise in, such as data protection or research software engineering.
+The sessions are a resource for community or organisation members to ask questions, receive targeted advice and support, and create space to start co-working or futher collaboration.
+The sessions are also an opportunity to learn about and connect with new teams and individuals that an attendee may not regularly connect with.
+
+```{figure} ../../figures/research-communities.*
+---
+height: 400px
+name: research-communities
+alt: Cartoon-like sketch of people in different nodes connecting into a central node with a robot in the middle. The people represent researchers in different research domains. One group is building a rocket ship, another group is fixing a car, and another group is working on a hot air balloon. In the central node, researchers are performing different tasks to build a robot. Some of researchers are taking the findings from the central node to the other nodes.
+---
+_The Turing Way_ project illustration by Scriberia. Used under a CC-BY 4.0 licence. DOI: [10.5281/zenodo.3332807](https://doi.org/10.5281/zenodo.3332807).
+```
+
+## Attending a drop-in session
```

Comment:
I think we might want to add in a bit here on what people will want to take into account when setting up a drop in session - how to promote to desired audiences, what the concerns or barriers might be to attending, how to address them etc?

Reply:
Okay coming back to this - I'm wondering if we put all the generic advice up top then add a "case study" for the Turing infrastructure drop ins? Not a hill for me but think it might make a bit more sense?

Reply:
Yeah coming back to it this morning, I think you're right! 

Will keep the generic stuff up to and add a case study at bottom. I'll focus on the infra drop-ins because we have more public resources for that